### PR TITLE
Found bug with CSS style on domain scan [ch17202]

### PR DESCRIPTION
**Pull request by alexander-medved**

### CODE REVIEW

File: scan.sh [hunk 1]
```diff
@@ -36,6 +36,10 @@ function report() {
         echo "Discovered URLs:" | tee -a scan.log.txt
         cat aquatone_urls.txt | tee -a scan.log.txt
         chmod -R go=rX,u=rwX ./*
+        sed -i -e 's/<link.*/<link href="https:\/\/stackpath.bootstrapcdn.com\/bootswatch\/4.1.3\/darkly\/bootstrap.min.css" \
+rel="stylesheet" integrity="sha384-w+yWASP3zYNxxvwoQBD5fUSc1tctKq4KUiZzxgkBSJACiUp+IbweVKvsEhMI+gz7" \
+crossorigin="anonymous">/g' aquatone_report.html 
+# We shuld delete the hack above when here: https://github.com/michenriksen/aquatone the source of stylesheet will be fixed!
```
Comment (max-lobur) [inline scan.sh]: please create a bug in michenriksen/aquatone repo

Comment (max-lobur) [reply scan.sh]: and add bug number here for ref

Comment (alexander-medved) [reply scan.sh]: done:
https://github.com/michenriksen/aquatone/issues/155

Comment (max-lobur) [reply scan.sh]: to src code :)

Comment (alexander-medved) [reply scan.sh]: done here https://github.com/verygood-ops/domain-scan/pull/3
@max-lobur Please marge in master.

**Comment by alexander-medved** [comment]
closed but https://github.com/verygood-ops/domain-scan/pull/3

**Pull request closed without merging**